app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  def update_users_posts_counter
+    user.increment(:posts_counter)
```

Comment:
- [ ] Kindly fix this typo error as the function is called `increment!`, not `increment`. The function `increment` has different usage and parameters.